README.md

```diff
@@ -18,6 +18,46 @@ These are examples of valid exercise names:
 - `javascript-2-b` : second part of exercise B for JavaScript.
 - `ruby-1-b` : first part of exercise B for Ruby.
 
+### Test messages
+
+Solving exercises that are part of the experiment is different from the regular website's process in two ways:
+
+1. Exercises are solved exclusively using in-browser coding.
+1. The test suite used to test the code is not available to the user.
+
```

Comment:
How about adding something here saying something like:

----

This means that when the code does _not_ pass one of the tests, the user won't see the code of the test that failed, or know with what input the code was called. To fix this, we need to provide a bit of context when we show errors to the user. 

For an assert_equal failure, the error with context might look as follows:

> We tried running `MyTests.MyTestMethod(1)` but got the following error:
> ```
> Expected: "foobar"
>   Actual: "barfood"
> ```